### PR TITLE
docs(core): update links to schema.json of @schematics/angular

### PR DESCRIPTION
**Pull request by mchl18**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: https://github.com/angular/angular/issues/39796

## What is the new behavior?

Links point to version 11 instead of version 7 blobs.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/schematics-authoring.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ For example, the hypothetical "Hello World" schematic might have the following s
 </code-example>
 
 
-You can see examples of schema files for the Angular CLI command schematics in [`@schematics/angular`](https://github.com/angular/angular-cli/blob/7.0.x/packages/schematics/angular/application/schema.json).
+You can see examples of schema files for the Angular CLI command schematics in [`@schematics/angular`](https://github.com/angular/angular-cli/blob/11.0.x/packages/schematics/angular/application/schema.json).
```
Comment (alan-agius4) [inline aio/content/guide/schematics-authoring.md:92]: Can you actually change this to point to the master branch instead?

```suggestion
You can see examples of schema files for the Angular CLI command schematics in [`@schematics/angular`](https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/application/schema.json).
```

**Comment by gkalpak** [comment]
Drive-by comment:

Please add `Fixes #39796` at the bottom of the commit message. (See our [commit message guidelines](https://github.com/angular/angular/blob/1d9e56eba06958e37ef8062ae7b65afa85f90632/CONTRIBUTING.md#commit) for more details.)

File: aio/content/guide/workspace-config.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Individual schematics for the default Angular CLI `ng generate` sub-commands are
 Specify the schematic name for a subcommand in the format `schematic-package:schematic-name`;
 for example, the schematic for generating a component is `@angular:component`.
 
-The JSON schemas for the default schematics used by the CLI to generate projects and parts of projects are collected in the package [`@schematics/angular`](https://github.com/angular/angular-cli/blob/7.0.x/packages/schematics/angular/application/schema.json).
+The JSON schemas for the default schematics used by the CLI to generate projects and parts of projects are collected in the package [`@schematics/angular`](https://github.com/angular/angular-cli/blob/11.0.x/packages/schematics/angular/application/schema.json).
```
Comment (alan-agius4) [inline aio/content/guide/workspace-config.md:84]:
```suggestion
The JSON schemas for the default schematics used by the CLI to generate projects and parts of projects are collected in the package [`@schematics/angular`](https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/application/schema.json).
```

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Thanks for this. See above comment please.

Also kindly add `closes #39796` as the commit message footer.

Thanks again.

**Review by aikithoughts — COMMENTED** [review]
Thanks for this! Can you change the link from 11.0.x to `master`? That way, it will remain (more) up to date.

**Comment by mchl18** [comment]
@aikidave @alan-agius4 updated the link and commit message, force-pushed.

**Review by alan-agius4 — APPROVED** [review]
Thanks for this. LGTM.

**Review by aikithoughts — APPROVED** [review]
Thank you!

Reviewed-for: global-docs-approvers

**Pull request closed without merging**